style.css

```diff
@@ -666,3 +664,421 @@ textarea {
   justify-content: center;
   align-items: flex-end;
 }
+
+@media only screen and (min-width: 768px) {
+  header {
+    width: 100%;
+    height: auto;
+    margin-bottom: 0;
+    position: sticky;
+    top: 0;
+  }
+
+  .toolbar {
+    width: 100%;
+    padding: 5px 100px;
+  }
+
+  .menu-placeholder {
+    display: none;
+  }
+
+  .desktop-navigation {
+    margin-left: auto;
+  }
+
+  .desktop-menu-placeholder {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 15px;
+  }
+
+  .desktop-nav-list > a {
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .md-light {
+    color: #fff;
+  }
+
+  .headline {
+    margin-top: 0;
+    width: 100%;
+    height: auto;
```

Comment:
[OPTIONAL] Great job with the portfolio, I think using` height: 100vh` , instead of `auto `will make our headline section stand out and will be more similar to the figma design 👍